AppServices/Validators/ValidatorExtensions.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Linq;
+
+namespace AppServices.Validators
+{
+    public static class ValidatorExtensions
+    {
+        public static bool isValidCpf(string cpf)
+        {
```

Comment:
Nomes de classe/métodos seguir padrão PascalCase
```suggestion
        public static bool IsValidCpf(string cpf)
        {
```